src/plugins/core/borders.ts

```diff
@@ -412,7 +412,7 @@ export class BordersPlugin extends CorePlugin<BordersPluginState> implements Bor
       if (
         border?.left &&
         col > 0 &&
-        !deepEquals(this.getCellBorder({ sheetId, col: col - 1, row })?.right, border?.left)
+        !deepEquals(this.getRawCellBorder({ sheetId, col: col - 1, row })?.right, border?.left)
```

Comment:
```suggestion
        !deepEquals(this.borders[sheetId]?.[col-1]?.[row]?.right, border?.left)
```
Even better, no ? Don't have to create a function that has to be named, required a comment on why we do not use deepCopy in this case, ...

What do you think ?

Reply:
Yes of course,

why make it simple when you can make it complicated ;)